### DISCUSSION TITLE
Issue with installing

### DISCUSSION THREAD

**Discussion by josroestberkel** [fabianishere/udm-iptv#350] [General]
Having problems to install the udm-iptv on unifi-os.

Works well for serveral months. But after the last unifi update, can i no longer install it properly

How do I get KPN IPTV working again?

![Naamloos2](https://github.com/user-attachments/assets/80eebcc1-035b-43dd-9ed9-57dcf4451036)
![Naamloos1](https://github.com/user-attachments/assets/7a9077e8-a641-409d-a74b-0d9e65b771bf)

**Comment by Rinus-86** [comment]
Did you fix the issue already? Because I have got the same problem now.

**Comment by Rinus-86** [comment]
@josroestberkel I fixed the issue of the masked services. If you run the command "sudo systemctl unmask udm-iptv.service
" you can start the services after that with the command udm-iptv restart. The erro should then be gone.

**Reply by josroestberkel** [reply]
Thanks! It works again after running that command.

**Comment by dmutsaers** [comment]
> But after the last unifi update, can i no longer install it properly.

Which update?
